### DISCUSSION TITLE
Newbie: Login into Supabase Studio Question

### DISCUSSION THREAD

**Discussion by Yakito** [supabase/supabase#4226] [Questions]
Hi all! super new to Supabase!

I followed the guide here https://supabase.com/docs/guides/hosting/docker to install Supabase on my host which went super smooth.

The issue I am having which I couldn't find an answer to is that after completing the installation basically anyone with my server address can access the backend.

I can't find a way to create user access to Supabase Studio.

Also, when trying to find a solution I saw some screenshots where there is a "settings" icon, which I don't seem to have. Below is a screenshot of my screen.

![image](https://user-images.githubusercontent.com/992405/144332677-4a5436d3-833e-4474-bec1-420ca899cf88.png)

Any suggestions will be appreciated! 
thanks!

**Comment by jet10000** [comment]
I change port from 3000 to 13000, open browser and visit http://127.0.0.1:13000 for me

```
## Studio port
STUDIO_PORT=13000
```

**Reply by Yakito** [reply]
Thanks! I just changed the `.env` file to match 13000 as the Studio port, and again, I am able to get into the Studio, but anyone with the URL can too.
Also, no signs of the settings icon.

When you go into https://URL:13000 it ask you for a user/pass ?
Thanks again!

**Reply by eliteksolutions** [reply]
I have the same problem

**Reply by Yakito** [reply]
Also, I can not create new projects, are you able to do that out of a fresh docker install?

**Reply by zephleggett** [reply]
Same issue here; I'm not sure auth though the ui is available in studio yet.

**Reply by doracl** [reply]
I have the same problem

**Comment by samerbahri98** [comment]
As of today, this bug is still unfixed. And it's really dangerous to self host supabase studio if this bug still exists.

**Reply by kiwicopple** [reply]
Answered in your GitHub issue: https://github.com/supabase/supabase/issues/4934

Ultimately, how you secure the Dashboard is up to you. We may look into a simple username/password by default, but you will still want to lock down access to everything in the docker set up

**Reply by Saif-Hossain-Mredul** [reply]
I am using supabase self host. I changed the DASHBOARD_USERNAME and DASHBOARD_PASSWORD, restarted the service, but still cant login. It is accepting the default username and password, but the changed one is not working.
